src/components/TodoHistoryListModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+
+import { useSelector } from "react-redux";
+import { AppState } from "src/store/rootReducer";
+
+import { colors, getShadow } from "src/styles";
+import { css, StyleSheet } from "aphrodite";
+import { i18n } from "src/locale";
+
+import { ITodo } from "src/types";
+
+import { StyledText } from "./StyledText";
+import { HistoryTodo } from "./Todo";
+
+export interface Props {}
+
+export const TodoHistoryListModal: FC<Props> = () => {
```

Comment:
this component is very similar to `TodoList` so I think u could merge it into one `TodoList` wich will receive `list` prop from the parent